docroot/themes/custom/mass_theme/templates/misc/status-messages.html.twig

```diff
@@ -21,8 +21,9 @@
  *   - class: HTML classes.
  */
 #}
-
+{{ attach_library('seven/classy.messages') }}
```

Comment:
Should mass_theme now be dependant somehow on seven theme? Does Olivero do that?

Reply:
We had that dependency when we switched to drupal 10, that's why we brought seven theme as a contrib to our D10

Reply:
OK, I guess we can keep it as an undeclared dependency.